compiler/q-implant/src/QImplant.cpp

```diff
@@ -287,19 +288,26 @@ void QImplant::write(loco::Graph *g)
 
 void QImplant::forward_qparam(loco::Graph *g)
 {
-  std::set<luci::CircleOpcode> forwardable_opcode{
-    luci::CircleOpcode::RESHAPE, luci::CircleOpcode::SPLIT, luci::CircleOpcode::TRANSPOSE};
+  /*
+   * TODO: add comment about how to add to the set
+   *
+   * If the operator doesn't change input tensor value,
+   * (Operator don't change input quantization parameter to output tensor)
+   * the operator's quantization parameter can be forwarded
+   */
+  std::set<luci::CircleOpcode> forwardable_opcode;
+  forwardable_opcode.emplace(luci::CircleOpcode::RESHAPE);
+  forwardable_opcode.emplace(luci::CircleOpcode::SPLIT);
```

Comment:
Qparam has to be forwarded to CircleSplitOut, which is the output of CircleSplit.

```suggestion
  forwardable_opcode.emplace(luci::CircleOpcode::SPLIT);
  forwardable_opcode.emplace(luci::CircleOpcode::CIRCLESPLITOUT);
```

Reply:
LGTM after this change.

Reply:
Reflected the suggestion!